sohbaker/java-tic-tac-toe
# Feedback

- update README to include how to test and how to run program
- ensure members of a class are private 
- refer to constant PLAYER_MARKS rather than the marks themselves
- use primitives where possible
- update test naming to reflect current test unit version
- use before block for test setup
- resolve type method error
- use more expressive naming
- use for loop instead of while
- pass IO as dependencies 
- remove errant file
